cluster/manifests/appdynamics/daemonset.yml

```diff
@@ -0,0 +1,53 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: appdynamics-machineagent
+    version: v1
+  name: appdynamics-machineagent
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      name: appdynamics-machinenagent
+      labels:
+        application: appdynamics-machineagent
+    spec:
+      hostNetwork: true
+      containers:
+      - name: appdynamics-machineagent-prod
+        image: pierone.stups.zalan.do/monitoring-appdynamics/appdynamics-machine-agent-4.2:latest
```

Comment:
Something like `4.2-1` where the first part is the app-dynamics version and the second part (after the dash) is a counter could make sense as a version.